bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartGraphs.java

```diff
@@ -29,350 +45,660 @@
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.ToolBar;
 
-import net.tourbook.Messages;
-import net.tourbook.application.TourbookPlugin;
-import net.tourbook.common.action.ActionOpenPrefDialog;
-import net.tourbook.common.font.MTFont;
-import net.tourbook.common.tooltip.ToolbarSlideout;
-import net.tourbook.common.util.Util;
-import net.tourbook.preferences.PrefPageAppearanceTourChart;
-import net.tourbook.tour.TourManager;
-
 /**
  * Tour chart properties slideout.
  */
 public class SlideoutTourChartGraphs extends ToolbarSlideout {
```

Comment:
* **Custom Track** in the tooltip `Show/hide Custom Track -> ...` should be lower case as **ANY** other graphs

*  The EF graph is selected in the toolbar (blue border) but not displayed, the other 2 graphs are not selected in the toolbar but displayed


![mt-cust-graph](https://user-images.githubusercontent.com/1283445/108619840-d883e080-7427-11eb-9acb-2079e6430d65.png)